.github/dependabot.yml

```diff
@@ -63,6 +63,7 @@ updates:
         - /modules/redpanda
         - /modules/registry
         - /modules/scylladb
+        - /modules/socat
```

Comment:
not a blocker, but IMO adding it to `core` will be great as a utility container implementation.

Reply:
@eddumelendez could you elaborate a bit? I started with that in mind but considered the modules are easier to consume, dependencies wise. But I'm not opposed to sing it to them core

Reply:
In this case, the socat module just uses the stdlib so all the pulled packages would already be there when using tc-go.

I understand the benefits for having it in the core, although with the current project layout would be more difficult to add it directly in the core, without a package to enclose the functionality. I.e. not using the same APIs for running it with the `socat.Run` function. If in the core, we should:
1. Create a socat package, which is basically the same as the module
2. Add it to the core but exposed as `testcontainers.RunSocat` which would lose a bit of the semantics above